.gitmodules

```diff
@@ -13,3 +13,6 @@
 [submodule "pkg/unvanquished_src.dpkdir"]
 	path = pkg/unvanquished_src.dpkdir
 	url = https://github.com/UnvanquishedAssets/unvanquished_src.dpkdir.git
+[submodule "libs/glm"]
+	path = libs/glm
+	url = https://github.com/g-truc/glm
```

Comment:
This should not be the cause of your problem BUT it would be better if you add the trailing `.git`.

Reply:
let's check then

Reply:
It seems it was because I had not added `libs/glm` to the commit.